enabler/src/de/schildbach/pte/ShProvider.java

```diff
@@ -164,4 +168,50 @@ protected NearbyLocationsResult nearbyStationsById(final String id, final int ma
 		uri.append("&input=").append(normalizeStationId(id));
 		return htmlNearbyStations(uri.toString());
 	}
+
+	protected static final Map<String, Style> STYLES = new HashMap<String, Style>();
+
+	static {
+		// Busse Kiel
+		putKielBusStyle("1", new Style(Style.parseColor("#7288af"), Style.WHITE));
```

Comment:
On the [official plan](http://www.netzplan-kiel.de/), the lines are both marked as rectangles (in the marks at the end points of each line) and hexagons (on the lines themselves), so I don't know what I should choose there.